migration/sql-test-files/048-unique-idx-failed-insert.sql

```diff
@@ -13,4 +13,4 @@ VALUES
    (
       now(), now(), '0a24d3c2-e0a6-4686-8051-ec0ea1915a28', '86af5178-9b41-469b-9096-57e5155c3f31', now(), now(), 'iteration test one', 'description', 'new', '/'
    )
```

Comment:
ok, I think it was just a blank line that was removed at the end of the file. My apologies.